scammer-label-propagation/src/preprocessing/get_data.py

```diff
@@ -0,0 +1,311 @@
+import time
+import torch
+import logging
+import requests
+import numpy as np
+import pandas as pd
+from datetime import datetime
+
+from src.constants import attacker_bots, victim_bots, SIMULTANEOUS_ADDRESSES, MIN_NEIGHBORS
+from src.storage import get_secrets
+
+logger = logging.getLogger(__name__)
+
+
+def get_all_related_addresses(central_node) -> str:
+    """
+    Querying allium. Returns the list of addresses that are first order neighbours of the central node.
+    :param central_node: str Address that will be the center of the graph
+    :return: str All addresses that are first order neighbours of the central node comma separated in a string
+    """
+    logger.info(f'{central_node}\tQuerying all related addresses')
+    API_key = get_secrets()['jsonRpc']['ALLIUM']
+    api_url = f'https://api.allium.so/api/v1/explorer/queries/Q71VcKtUFjBtloXNZtpD/run'
```

Comment:
pls add a comment on what the underlying sql is

Reply:
Added it in a new file, as it was too long for a comment. Now all the sql queries are shown